spaceship/lib/spaceship/connect_api/tunes/tunes.rb

```diff
@@ -17,9 +17,13 @@ def tunes_request_client
         # ageRatingDeclarations
         #
 
-        def get_age_rating_declaration(app_store_version_id: nil)
-          params = tunes_request_client.build_params(filter: nil, includes: nil, limit: nil, sort: nil)
-          tunes_request_client.get("appStoreVersions/#{app_store_version_id}/ageRatingDeclaration", params)
+        def get_age_rating_declaration(app_info_id: nil, app_store_version_id: nil)
+          raise "Keyword 'app_store_version_id' is deprecated and 'app_info_id' is required" if app_store_version_id
+
+          if app_info_id
+            params = tunes_request_client.build_params(filter: nil, includes: nil, limit: nil, sort: nil)
+            tunes_request_client.get("appInfos/#{app_info_id}/ageRatingDeclaration", params)
+          end
```

Comment:
```suggestion
          raise "Keyword 'app_store_version_id' is deprecated and 'app_info_id' is required" if app_store_version_id || app_info_if.nil?

          params = tunes_request_client.build_params(filter: nil, includes: nil, limit: nil, sort: nil)
          tunes_request_client.get("appInfos/#{app_info_id}/ageRatingDeclaration", params)
```

This handles the case where both `app_store_version_id` and `app_info_id` are `nil` slightly better. The current will do nothing in that situation whereas my suggestion gives you the reason and then causes a crash.